<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SRDB" failures="3" tests="4" name="Hello" time="2.081" errors="0" timestamp="2021-06-30T13:43:46 IST">
  <testcase classname="selenium.testng.NoonSeleniumTestNG.MyTestClass4" name="testAlpha" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [20] but found [35]">
      <![CDATA[java.lang.AssertionError: expected [20] but found [35]
at selenium.testng.NoonSeleniumTestNG.MyTestClass4.testAlpha(MyTestClass4.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testAlpha -->
  <testcase classname="selenium.testng.NoonSeleniumTestNG.MyTestClass4" name="testBeta" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at selenium.testng.NoonSeleniumTestNG.MyTestClass4.testBeta(MyTestClass4.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testBeta -->
  <testcase classname="selenium.testng.NoonSeleniumTestNG.MyTestClass4" name="testCosmo" time="2.012">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method selenium.testng.NoonSeleniumTestNG.MyTestClass4.testCosmo() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method selenium.testng.NoonSeleniumTestNG.MyTestClass4.testCosmo() didn't finish within the time-out 2000
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testCosmo -->
  <testcase classname="selenium.testng.NoonSeleniumTestNG.MyTestClass4" name="tetsDelta" time="0.001"/>
</testsuite> <!-- Hello -->
